components/02-molecules/accordion/_accordion.scss

```diff
@@ -0,0 +1,94 @@
+@use '../../00-tokens/tokens';
+@use '../../01-atoms/atoms';
+
+// The accordion group.
+.accordion {
+  margin-bottom: var(--spacing-page-inner);
+  padding-block: var(--size-spacing-6);
+}
+
+.accordion__controls {
+  @include atoms.list-reset;
```

Comment:
This is a new way to reference these resets. Example, the breadcrumbs scss file uses the list/button separately, then `@include list.reset;` or `@include buttons.reset;` within the file.

I don't know which option is best but want to point out the inconsistency.

Reply:
Yeah, I wanted to get away from `@use`ing twelve different files for the various mixins and variables I needed. That's why the `@forward` is in each of the levels files `tokens.scss`, `atoms.scss`, etc. 

The breadcrumbs should be updated to use this method. I'll go ahead and do that now, but if you find any others, they should be done in separate tickets.

Reply:
Sounds good. Thanks for clarifying the intent here.